### ISSUE TITLE
Feature request: share button

### ISSUE THREAD

**Issue by jamesbraza** [1bl4z3r/hermit-V2#36]
It would be cool to support a share button at the bottom of posts. It could be an opt-in feature, or enabled by default.

https://discourse.gohugo.io/t/share-buttons-for-blog-posts/12384 is a forum post with some ideas, the best seems to be https://codingnconcepts.com/hugo/social-icons-hugo/.

Thanks for your consideration, let me know what you think

**Comment by 1bl4z3r** [comment]
Hi @jamesbraza

Share is something that I thought of implementing but decided against because it is complicated for no particular reason

**Comment by 1bl4z3r** [comment]
Hi again,

Can we reposition Share button from bottom of the page to bottom navigation bar?
Using my site as an example, it gonna look like this.
![image](https://github.com/1bl4z3r/hermit-V2/assets/34515568/dad514c8-1d3d-423f-ba3b-5119c34a1f62)
In desktop, hovering on the button will open modal for the share links.
However, in mobile, I haven't found a good place to put that yet, so open for suggestions.

Let me know if you like this

**Comment by jamesbraza** [comment]
Oh yeah, at the bottom of the page works too!  You're right, it seems to make more sense there.

Good thinking on the modal too, like it.

**Comment by 1bl4z3r** [comment]
Cool cool cool
I will try to make something which follows Hermit styles as closely as possible.

**Comment by 1bl4z3r** [comment]
Update:
I got icon svg and location dialed down so that it looks coherent with the overall theme and not be an eyesore. Let me know if you have further suggestions.

In desktop, it will be at the extreme right hand.
![image](https://github.com/1bl4z3r/hermit-V2/assets/34515568/beccb83d-9031-4bcb-a8d3-4869a5dd5f10)

In mobile, it will be on the left of menu flyout
![image](https://github.com/1bl4z3r/hermit-V2/assets/34515568/1705d088-5c4a-4410-a176-9ce44b3f6435)

**Comment by 1bl4z3r** [comment]
Update : I suck at styling anything.

What have I accomplished?

A new partial is added which will be invoked if site param is set. I have added Linkedin, X(Twitter), Email and Facebook

In desktop, it will be shown at extreme left.
![image](https://github.com/1bl4z3r/hermit-V2/assets/34515568/14e9002b-b4d6-411c-a1af-d676f9726fe7)

In mobile, it will also be on extreme left
![image](https://github.com/1bl4z3r/hermit-V2/assets/34515568/054b2da9-0725-412e-aa70-1ae7668cd049)

Have I used 100% of my brain capacity. Nope
